tests/queries/0_stateless/02313_group_by_modifiers_with_non-default_types.reference

```diff
@@ -0,0 +1,113 @@
+-- { echoOn }
```

Comment:
Looks like this file was added by mistake, since there is no corresponding .sql file.

Reply:
Yes, thanks